AlgebraInLean/World0LeanIntroSOLUTIONS/Sheet1Solutions.lean

```diff
@@ -0,0 +1,88 @@
+/-
+This is a solutions sheet.
+-/
+
+import Mathlib.Tactic
+
+/-
+Welcome!
+
+This is an interactive problem set, where you'll learn to use Lean 4,
+a theorem prover and programming language, while exploring the basics
+of abstract algebra. Through formalizing and proving theorems on groups,
+rings, and fields, you'll become comfortable with using Lean 4 to
+help you verify theorems in a user-friendly way.
+
+This problem set is written under the assumption that you, the user, have
+already completed the Natural Number Game: <https://adam.math.hhu.de/#/g/leanprover-community/nng4>
+Additionally, it is assumed that the user is in the process of taking
+(or is already familiar with) an abstract algebra class, so explanations
+of concepts will be more concise than a textbook's.
+
+There are solutions provided for your reference. Should you be interested in
+learning more about specific tactics, Lean 4 documentation is available online.
+
+Learn about all things Lean 4 here: <https://leanprover-community.github.io/index.html>
+Full documentation can be found here: <https://leanprover-community.github.io/mathlib4_docs/>
+
+****
+
+Before we dive into abstract algebra in World 1, we will begin with the basics
+of Lean in World 0. If you are already familiar with using Lean, feel free to
+skip World 0 and dive straight into World 1. Additionally, if on any sheet you
+feel comfortable with a tactic, feel free to skip exercises and move on to learning
+the next one, as some exercises can be repetitive.
+
+As you saw in the Natural Number Game, in Lean, theorems are proven using the help
+of tactics. World 0 will familiarize you with a few basic tactics through some
+exercises concerning logic and basic arithmetic.
+
+To get you used to reading Lean, let us begin with a very basic exercise from
+the Natural Number Game.
+--/
+
+example : 37 * x + q = 37 * x + q := by
+  rfl
+
+/-
+The above example is the very first level of the Natural Number Game.
+In the Natural Number Game, you typed a tactic into a text box, and
+the active goals updated. Here, it is a little different.
+
+Putting your cursor after the word "by," you should see the Tactic State
+appear in your Lean Infoview window to the right of your screen.
```

Comment:
Maybe add something saying how to pull up the infoview if they don't have it up already.